docs/AR.md

```diff
@@ -0,0 +1,131 @@
+    - _Rolloff model:_ The rolloff model which determines how the sound volume will fall off between minDistance and maxDistance. Can take `None`, `Linear` & `Logarithmic` values as String
+- **MP4:**
+  - _Chroma key filtered video_ - If the video you want to add to the Augmented Reality scene has a background colour that you want to delete, add the colour code in HEX format here.
+
+## Display of added tourstopps in the app
+
+To download the 3D Model from the Tour detail screen:
+
+1. Press the hamburger menu button in the top right corner of the app and select "Tours and Places" from the drop-down menu,
+2. Scroll to the bottom of the page and in the "Tours" category select the section where you have added Tourstopps,
+3. Select the tour you added Tourstopps to from the displayed tour list and scroll to the bottom of the tour details page,
+4. Select the downloaded file from the Artworks list and click on the "Downloaden & AR Kunst gucken" button and wait for the 3D object to download,
+5. Click the View AR Art button and view the 3D model on the screen that opens.
+
+To download the 3D Model from the settings screen:
+
+- **Note:** In order to avoid an error on the settings page, please open an object called `ar` in the `settings` object in `globalSettings` and write the id of the tour you added tourstopp with the key `tourId`.
+
+1. Go to the settings page of the application.
+2. Select `AR Settings`.
+3. Download the "Sample File" to your device.
+
+To delete the 3D Model from the device:
+
+1. Open the detail page of the relevant tour.
+2. Scroll down the detail page.
+3. Click on the "Kunstwerke laden" button and wait for the modal to open.
+4. In the opened modal, an x will be displayed next to the items you have downloaded to your device
+5. Click on the downloaded item. Select "Delete" on the warning screen.
+6. If you see the download button again, the 3D object has been deleted from your device.
+
+Delete the 3D Model from the settings screen:
+
+1. Go to the settings page of the app.
+2. Select "AR settings".
+3. In the list you will see an "x" next to the items you have downloaded to your device.
+4. Click on the downloaded item. Select "Delete" on the warning screen.
+5. When you see the download button again, the 3D object has been deleted from your device.
+
+## Converter required when uploading a 3D Model
+
+- [**Download FBX to VRX converter**](https://fileserver.smart-village.app/development/augmented-reality/VRX_Converter/bin/ViroFBX)
+- [ViroFBX Documentation (MacOS)](https://github.com/ViroCommunity/ViroFBX)
+- [ViroFBX Documentation (Linux)](https://github.com/ViroCommunity/ViroFBX/tree/linux-support)
+
+**How to use Converter with Images**
+
+![](./assets/VRXConverter/VRXConverter1.png)
+![](./assets/VRXConverter/VRXConverter2.png)
+![](./assets/VRXConverter/VRXConverter3.png)
+![](./assets/VRXConverter/VRXConverter4.png)
+
+**How to use Converter with Video**
```

Comment:
How to use converter (video)

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/659/commits/e29043d4a78964b955874ece2ce58b818c237f9b